docs/getting_started.rst

```diff
@@ -54,6 +54,49 @@ Connecting with a client
     # Disconnect from the server
     client.disconnect()
 
+**************
+Authentication
+**************
+
+When constructing a ``Cloudant`` client, you can authenticate using the
+[cookie authentication](http://guide.couchdb.org/editions/1/en/security.html#cookies) functionality.
+If you need the cookie to automatically renew when it expires,
+construct the client using ``auto_renew=True``:
+
+.. code-block:: python
+
+    # Create client using auto_renew to automatically renew expired cookie auth
+    client = Cloudant(USERNAME, PASSWORD, url='https://acct.cloudant.com',
+                     connect=True,
+                     auto_renew=True)
+
```

Comment:
The default value for `auto_renew` is `False`.  Would something like this suffice?
```
This library can be used in app server environments (i.e. django).
When connecting to ``Cloudant`` or ``CouchDB``, you have the option of using the
``connect=True`` argument during ``client`` construction or by calling ``client.connect()``.
If the app server requires automatic renewal of the cookie authentication, add
the ``auto_renew=True`` argument when constructing the ``client``.
When disconnecting, use ``client.disconnect()`` to properly logout and end a client session.
```
This would be further down in the documentation so the user would already have seen how to construct a `client`.